### PR TITLE
Loot rework part 2.

### PR DESCRIPTION
**Pull request by Evankhell561**
## About The Pull Request
See changelog.

## Why It's Good For The Game
Kotmap.

## Changelog
:cl:
add: a chemical that can rejuvenate objects.
refactor: oldificator code
tweak: the probability of finding old items in trash piles increases to 80.
tweak: the price of the items affects their spawn frequency.
/:cl:

### CODE REVIEW

**Review by genessee596 — APPROVED** [review]

File: code/modules/reagents/recipes.dm [hunk 1]
```diff
@@ -2096,3 +2096,8 @@
 	result = "nanovoice"
 	required_reagents = list("uncap nanites" = 1, "synaptizine" = 1)
 	result_amount = 1
+
+/datum/chemical_reaction/rejuvenating_agent
+	result = "rejuvetaning_agent"
```
Comment (MemedHams) [inline code/modules/reagents/recipes.dm:2101]: ah yes, rejuvetaning agent

**Comment by TheShown911** [comment]
conflicts

**Review by Hopekz — CHANGES_REQUESTED** [review]
Ping me when conflicts are resolved for a final check and merge

**Comment by Evankhell561** [comment]
> Ping me when conflicts are resolved for a final check and merge

Ready.

**Comment by Hopekz** [comment]
> Ready.

Sorry;
One more in code/modules/mechs/equipment/combat.dm

**Review by genessee596 — APPROVED** [review]
i still approve

**Review by Hopekz — APPROVED** [review]
was a long review but I managed to review it.
Good for merge once the conflict is resolved

**Comment by Evankhell561** [comment]
> was a long review but I managed to review it.
> Good for merge once the conflict is resolved

ready

**Pull request merged**